utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -27,7 +27,7 @@ class FirebaseEventLogger(
   /** Logs an event to Firebase Analytics with [NETWORK_USER_PROPERTY] and [COUNTRY_USER_PROPERTY]. */
   override fun logEvent(eventLog: EventLog) {
     bundle = eventBundleCreator.createEventBundle(eventLog)
-    firebaseAnalytics.logEvent(eventLog.actionName.toString(), bundle)
+    firebaseAnalytics.logEvent(eventLog.context.activityContextCase.toString(), bundle)
```

Comment:
Should this be using the enum's ``name`` property for portability?

Reply:
done.